src/pykx/pandas_api/pandas_meta.py

```diff
@@ -209,7 +208,22 @@ def abs(self, numeric_only=False):
         if numeric_only:
             tab = _get_numeric_only_subtable(self)
         return q.abs(tab)
-
+    
+    @api_return
+    def isin(self, values):
+        tab = self
+        dic_value = kx.q("{$[98h = type x; flip x; x]}", values)
+        return q("{flip x! {"
+                    "tipo: abs[type z[x][0]];"
+                    "y: $[99h = type y; y[x]; y];"
```

Comment:
Minor: You use several times brackets when they are not necessary, is this an standard? For example here you could have written `y x`